frame/lending/src/currency.rs

```diff
@@ -194,6 +194,14 @@ pub mod defs {
 }
 
 pub use defs::*;
+use primitives::currency::ValidCurrency;
 use scale_info::TypeInfo;
 
 pub type CurrencyId = u128;
+pub struct AllValidCurrencyId;
+impl ValidCurrency<CurrencyId> for AllValidCurrencyId {
+	fn valid_currency_id(_currency_id: CurrencyId) -> bool {
+		// all other assets in mock are valid
```

Comment:
I would prefer mocks reuse some trait from helpers and also do not allow 0 for consistency of tests and runtime